blockchain/options.go

```diff
@@ -3,70 +3,94 @@ package blockchain
 import "time"
 
 const (
-	defaultEthEndpoint        = "https://rinkeby.infura.io/00iTrs5PIy0uGODwcsrb"
+	defaultLivechainEndpoint  = "https://rinkeby.infura.io/00iTrs5PIy0uGODwcsrb"
 	defaultSidechainEndpoint  = "https://sidechain-dev.sonm.com"
-	defaultGasPrice           = 20000000000 // 20 Gwei
-	defaultGasPriceSidechain  = 0
+	defaultLivechainGasPrice  = 20000000000 // 20 Gwei
+	defaultSidechainGasPrice  = 0
 	defaultBlockConfirmations = 5
+	defaultLogParsePeriod     = time.Second
 )
 
+// chainOpts describes common options
+// for almost any geth-node connection
+// (live Eth network, rinkeby, SONM sidechain
+// or local geth-node for testing).
+type chainOpts struct {
+	gasPrice           int64
+	endpoint           string
```

Comment:
its incorrect to includes params to connect to rpc and params to interact with blockchain in one struct

Reply:
why exactly